05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,32 @@
-.lawn {
-    width: 100%;
-    height: 300px;
-}
+td {
+  width:500px;
+  padding:0px;
 
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
 }
 
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+td + td {
```

Comment:
I hope you know what this [means](http://www.w3schools.com/cssref/sel_element_pluss.asp)